### ISSUE TITLE
The RTMP output does not work is a gray video on periscope

### ISSUE THREAD

**Issue by streamingPer** [begeekmyfriend/yasea#513]

**Comment by begeekmyfriend** [comment]
What is the device? Phone or pad?

**Comment by streamingPer** [comment]
Galaxy S6

**Comment by begeekmyfriend** [comment]
What about other phone?

**Comment by streamingPer** [comment]
S7 Edge and s4

**Comment by streamingPer** [comment]
didnt work also could you tell me how to solve it because on youtube and facebook works fine. i think it bout key frame? would you tell me what's the problem

**Comment by begeekmyfriend** [comment]
I do not think that is the issue of yasea. Why not ask Periscope staff for help?

**Comment by streamingPer** [comment]
How to edit keyframe?

**Comment by begeekmyfriend** [comment]
Key frames can be edited [here](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsFlvMuxer.java#L876). But I have never tried it.

**Comment by streamingPer** [comment]
yes i trying soft encoding, but don't work in periscope

**Comment by streamingPer** [comment]
thank u si :)r
